### DISCUSSION TITLE
Removing Elements from the UI

### DISCUSSION THREAD

**Discussion by Daviid1010** [requarks/wiki#5234] [Help / Questions] [answered]
Hi there,
I am creating a version of wiki js for my use. I want to remove some parts of the UI. Such as the tagging, author info, and comments section. Is there a way I can do that either as an admin user or editing the code?

**Comment by NGPixel** [comment, marked as answer]
You can already remove the comments section by disabling "Comments" from the admin > General section.

As for the other parts, this is coming in v3. In the meantime, you'd have to use CSS to remove them. There're some posts providing examples.

**Reply by Daviid1010** [reply]
Thanks for the reply! Oh is it just some css overrides for the moment? I'll have a look!
If you had it handy could u link those posts?
Great app though fantastic UI!
